src/Raven.Server/Documents/Indexes/IndexDefinitionBaseServerSide.cs

```diff
@@ -180,16 +180,18 @@ public static class IndexVersion
             public const long CoraxSearchWildcardAdjustment_60 = 60_006; // RavenDB-22937
 
             public const long Base61Version = 61_000;
-            
+            public const long Base62Version = 62_000;
```

Comment:
In practice the above `Base61Version = 61_000;` is base for 6.2 version. Since we decided to not release 6.1 and converted it to 6.2.

So the right thing to do here would be:
- rename `Base61Version` to `Base62Version`